src/libmongoc/src/mongoc/mongoc-ocsp-cache.c

```diff
@@ -174,10 +174,8 @@ _mongoc_ocsp_cache_get_status (OCSP_CERTID *id,
 
    *cert_status = entry->cert_status;
    *reason = entry->reason;
-   *this_update =
-      ASN1_item_dup (ASN1_ITEM_rptr (ASN1_TIME), entry->this_update);
-   *next_update =
-      ASN1_item_dup (ASN1_ITEM_rptr (ASN1_TIME), entry->next_update);
```

Comment:
This is to fix leaks on ASAN that popped up after client side encryption tests started caching responses.

`_mongoc_ocsp_tlsext_status` expects to retrieve the `this_update` / `next_update` pair from the cache or from `OCSP_resp_find_status` and treats both the same. Since the `this_update` / `next_update` from `OCSP_resp_find_status` must not be freed, I changed this to behave the same way.